src/main/resources/reference.conf

```diff
@@ -1348,11 +1348,14 @@ resourceTypes = {
       approve = {
         description = "Approve or reject the snapshot builder request"
       }
+      set_parent = {
+        description = "Enables setting the parent on the request object"
+      }
     }
     ownerRoleName = "owner"
     roles = {
       owner = {
-        roleActions = ["get", "update", "delete"]
+        roleActions = ["get", "update", "delete", "set_parent"]
```

Comment:
This is the fix, but is also the security hole, correct?

Reply:
Yes. https://broadinstitute.slack.com/archives/C03GMG4DUSE/p1715026844061349 discusses the real solution.